### PR TITLE
[ET-1839] Show series pass header on series page ticket form.

### PR DESCRIPTION
**Pull request by rafsuntaskin**
### 🎫 Ticket

[ET-1839] 

### ✔️ Checklist
- [x] I've included a changelog entry in the readme.txt file. 
- [x] My code is tested. 
- [x] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-1839]: https://stellarwp.atlassian.net/browse/ET-1839?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tests/ft_integration/Tribe/Tickets/__snapshots__/AttendeesTest__test_display__single event in series with Single Tickets, RSVPs and Series Passes__0.snapshot.html [hunk 1]
```diff
@@ -84,7 +84,7 @@ <h3 class="tec-tickets__admin-attendees-overview-title">
 							#POST_ID						</span>
 					</div>
 					<div class="tec-tickets__admin-attendees-overview-ticket-type-list-item-stat">
-						3 issued (97 available)					</div>
```
Comment (lucatume) [inline tests/ft_integration/Tribe/Tickets/__snapshots__/AttendeesTest__test_display__single event in series with Single Tickets, RSVPs and Series Passes__0.snapshot.html:87]: Why these changes?

**Review by lucatume — CHANGES_REQUESTED** [review]
A question about the changes to the snapshots.

File: tests/ft_integration/Tribe/Tickets/__snapshots__/AttendeesTest__test_display__single event in series with Single Tickets, RSVPs and Series Passes__0.snapshot.html (hunk 1 above)
Comment (rafsuntaskin) [reply tests/ft_integration/Tribe/Tickets/__snapshots__/AttendeesTest__test_display__single event in series with Single Tickets, RSVPs and Series Passes__0.snapshot.html:87]: These are not related to the changes for the feature but to stabilize the tests. There were some random fails as the `menu_order` was not set for them.

Comment (rafsuntaskin) [reply tests/ft_integration/Tribe/Tickets/__snapshots__/AttendeesTest__test_display__single event in series with Single Tickets, RSVPs and Series Passes__0.snapshot.html:87]: They just moved places. No real data changed

**Review by lucatume — APPROVED** [review]
